Intro_markdown.Rmd

```diff
@@ -213,31 +209,31 @@ To know more about a package, it is always useful to read the associated documen
 
 It is important to be able to import data both from your own computer as well as the Analytical Platform cloud storage.
 
-### Importing data using a local version of RStudio
+### Importing data using a local version of RStudio 
 
-If you're using a local version of RStudio on your laptop (i.e. not accessed via the Analytical Platform control panel), you can import data from .csv files into Rstudio by clicking on the Environment tab and then the Import button. You can then navigate to the folder where the dataset "Offenders_Chicago_Police_Dept_Main.csv" is saved and click on it. A window will then appear which will include on the bottom right a preview of your data. Here it looks good, so we can click on import. 
+If you're using a local version of RStudio on your laptop (i.e. not accessed via the Analytical Platform control panel), you can import data from .csv files into RStudio by clicking on the Environment tab and then the Import button. You can then navigate to the folder where the dataset "Offenders_Chicago_Police_Dept_Main.csv" is saved and click on it. A window will then appear which will include on the bottom right a preview of your data. Here it looks good, so we can click on import. 
 
 You can now see by looking in the environment window that an object has been created (the offenders dataset), and that it has 1413 observations and 9 variables.
 
-Now look at the Console tab. You should see the commands library and read_csv() appear with the whole path to the data set. It is a good idea to copy and paste these commands inside your script, so you won't need to do this again to load the data.
+Now look at the Console tab. You should see the commands library and `read_csv()` appear with the whole path to the data set. It is a good idea to copy and paste these commands inside your script, so you won't need to do this again to load the data.
 
-Alternatively you can simply use the function read_csv() from the readr package (which is included as part of tidyverse - see section 2.2):
+Alternatively you can simply use the function `read_csv()` from the `readr` package (which is included as part of tidyverse - see section 2.2:
```

Comment:
Very minor, but looks like a bracket was removed from the end of this bit of text, and now means there's an open bracket with no matching close bracket.